olive/model.py

```diff
@@ -234,6 +237,52 @@ def prepare_session(self, inference_settings: Dict[str, Any], device: Device, ra
 
         return get_ort_inference_session(self.model_path, inference_settings)
 
+    def nodes(self):
+        all_nodes = []
+        for graph in self.graphs():
+            for node in graph.node:
+                all_nodes.append(node)
+        return all_nodes
+
+    def graph(self):
+        return self.load_model().graph
```

Comment:
>self.load_model().graph [](http://example.com/codeflow?start=15&length=23)

do we need to cache the graph to avoid loading the model multiple times when it is called multiple times.

Reply:
Yeah I'll cache it.